mso/resource_mso_schema_template_bd.go

```diff
@@ -618,110 +767,59 @@ func resourceMSOTemplateBDUpdate(d *schema.ResourceData, m interface{}) error {
 		dhcpPolMap = nil
 	}
 
+	dhcpPolList := make([]interface{}, 0)
```

Comment:
Does update work as expected when configuring multiple blocks of dhcp policies? -
Example -
Add blocks, Remove blocks one at a time etc.